src/scripts/components/widgets/tcr/ChallengePeriod.js

```diff
@@ -0,0 +1,202 @@
+/* @flow */
+import React, { Component } from 'react'
```

Comment:
What is this component? `ChallengePeriod.js` doesn't seem like the most self-explanatory name?